airflow/ui/src/pages/DagsList/Dag/Code/Code.tsx

```diff
@@ -55,13 +55,9 @@ export const Code = () => {
     data: code,
     error: codeError,
     isLoading: isCodeLoading,
-  } = useDagSourceServiceGetDagSource(
-    {
-      fileToken: dag?.file_token ?? "",
-    },
-    undefined,
-    { enabled: Boolean(dag?.file_token) },
-  );
+  } = useDagSourceServiceGetDagSource({
+    dagId: dagId ?? "",
```

Comment:
is version_number also needed?

Reply:
I left this to @bbovenzi and @pierrejeambrun to implement. I'm not sure how the version should be listed, selected and sent over through the API

Reply:
sounds good